tests/placeAndMakeSwiftPerpBankrun.ts

```diff
@@ -512,6 +518,193 @@ describe('place and make swift order', () => {
 		await takerDriftClient.unsubscribe();
 	});
 
+	it('should not fill against the vamm if the user is toxic', async () => {
+		const slot = new BN(
+			await bankrunContextWrapper.connection.toConnection().getSlot()
+		);
+
+		const [lookupTableInst, lookupTableAddress] =
+			AddressLookupTableProgram.createLookupTable({
+				authority: makerDriftClient.wallet.publicKey,
+				payer: makerDriftClient.wallet.publicKey,
+				recentSlot: slot.toNumber() - 10,
+			});
+
+		const extendInstruction = AddressLookupTableProgram.extendLookupTable({
+			payer: makerDriftClient.wallet.publicKey,
+			authority: makerDriftClient.wallet.publicKey,
+			lookupTable: lookupTableAddress,
+			addresses: [
+				SystemProgram.programId,
+				solUsd,
+				solUsdLazer,
+				...makerDriftClient
+					.getPerpMarketAccounts()
+					.map((account) => account.pubkey),
+				...makerDriftClient
+					.getPerpMarketAccounts()
+					.map((account) => account.amm.oracle),
+				...makerDriftClient
+					.getSpotMarketAccounts()
+					.map((account) => account.pubkey),
+				...makerDriftClient
+					.getSpotMarketAccounts()
+					.map((account) => account.oracle),
+				PYTH_LAZER_STORAGE_ACCOUNT_KEY,
+			],
+		});
+
+		const tx = new Transaction().add(lookupTableInst).add(extendInstruction);
+		await makerDriftClient.sendTransaction(tx);
+		console.log(`Lookup table: ${lookupTableAddress.toBase58()}`);
+
+		const [takerDriftClient, takerDriftClientUser] =
+			await initializeNewTakerClientAndUser(
+				bankrunContextWrapper,
+				chProgram,
+				usdcMint,
+				usdcAmount,
+				marketIndexes,
+				spotMarketIndexes,
+				oracleInfos,
+				bulkAccountLoader
+			);
+		await takerDriftClientUser.fetchAccounts();
+
+		// Create 11 subaccounts so our user is considered toxic
```

Comment:
ideally we dont have this test break if this logic updates, probably better to update the `disable_update_perp_bid_ask_twap` user stats field to true